### DISCUSSION TITLE
Slow start configuration

### DISCUSSION THREAD

**Discussion by anupam-meesho** [envoyproxy/gateway#4509] [Q&A]
Hi Team, 
Some background: We have our whole infra in kubernetes. We have configured contour as an ingress gateway for the means of routing traffic across the clusters.
Question: We have configured slow start for some of our services. We came across two different behaviours for the different pods. One is honouring the slow start other one is takes a way longer time to ramp up the pods to the full capacity. It would be very helpful if you can point to the certain documentation of code that governs this behaviour predictively. Below is the configuration we are using:

    slowStart:
        aggression: 0.2
        minPercent: 1
        window: 150s
        
   Also attached are the two different behaviours.
   <img width="848" alt="image" src="https://github.com/user-attachments/assets/96efaa62-a609-41db-9ae2-d6d159e5b54b">

**Comment by arkodg** [comment]
Hi @anupam-meesho this is the Envoy Gateway project, if you'd like to configure `slowStart` using this controller, you can set this config https://gateway.envoyproxy.io/docs/api/extension_types/#slowstart using the BackendTrafficPolicy https://gateway.envoyproxy.io/docs/tasks/traffic/load-balancing/
